bin/anthology/anthology.py

```diff
@@ -54,38 +54,35 @@ def import_directory(self, importdir):
         for xmlfile in glob(importdir + "/xml/*.xml"):
             self.import_file(xmlfile)
         self.pindex.verify()
-        
+
```

Comment:
This reminds me of something I've did rigorously but never made explicit: I used [black](https://black.readthedocs.io/en/stable/) to keep the formatting style consistent. Do you think it would be a good idea to add this to the README as "coding guidelines" or something?

Reply:
I do think this is a good idea and we could implement it.